README.md

```diff
@@ -1,22 +1,85 @@
-# cluster-logging-operator
+# Cluster Logging Operator
+
 An operator to support OKD aggregated cluster logging.  Cluster logging configuration information
 is found in the [configuration](./docs/configuration.md) documentation.
 
-## Quick Start
+# Hacking
+
+Note: In the rest of this doc "CLO" stands for Cluster Logging Operator
+
+## Preparation
+
+Clone this repository into $GOPATH/github.com/openshift/cluster-logging-operator
+
+Install `podman` and `skopeo`. For example on  Fedora:
```

Comment:
We don't use skopeo to push images anymore because using rootless podman stores all images under `$HOME/.local/containers` and AFAIK skopeo does not support rootless container storage. Anyway `podman push` does the work well enough!

Reply:
I thought that was the idea, but we still  have https://github.com/openshift/cluster-logging-operator/blob/master/hack/common#L185 - what's the equivalent podman command?

Reply:
I believe this function is not used. `hack/deploy-image.sh` and `olm_deploy/scripts/catalog-build.sh` were replaced to use `podman push`.

Reply:
> I thought that was the idea, but we still have https://github.com/openshift/cluster-logging-operator/blob/master/hack/common#L185 - what's the equivalent podman command?

It's ```podman image (pull, tag, push)``` which makes ```scopeo copy``` so much more intuitive.

Reply:
The only extra requirement that made skopeo previously mandatory was that of copy an image tag to a target registry and relabel this tag. To be precise, we used to build local tags like:
```
image-registry.openshift-image-registry.svc:5000/openshift/origin-cluster-logging-operator:latest
```
and by using skopeo pushing them as to the dev cluster registry as:
```
127.0.0.1:5000/openshift/origin-cluster-logging-operator:latest
```
However, since [PR-494](https://github.com/openshift/cluster-logging-operator/pull/494/files#diff-b67911656ef5d18c4ae36cb6741b7965R7) we use the latter tag for local tags.

In addition skopeo is not able to copy from a local oci storage used by a rootless podman.